### PR TITLE
chore: add docs related to sp exit and bucket migration

### PR DESCRIPTION
**Pull request by Fynnss**
## Description

Add docs related to sp exit and bucket migration

### CODE REVIEW

File: docs/guide/greenfield-blockchain/run-node/become-testnet-validator.md [hunk 1]
```diff
@@ -119,8 +119,10 @@ Replace the values in the following JSON and save it as create_validator_proposa
 }
 ```
 
-Submit the proposal. Ensure the validator account has enough BNB tokens.
+Ensure the all the account in the proposal has enough BNB tokens.
 ```bash
+
+./build/bin/gnfd tx sp create-storage-provider ../create_storage_provider.json --from funding  --node "https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443"
```
Comment (KeefeL) [inline docs/guide/greenfield-blockchain/run-node/become-testnet-validator.md:125]: We should delete step 4 Grant authorization and the next gnfd tx gov submit-proposal command.

Comment (KeefeL) [reply docs/guide/greenfield-blockchain/run-node/become-testnet-validator.md:125]: Use gnfd other than ./build/bin/gnfd

Comment (KeefeL) [inline docs/guide/greenfield-blockchain/run-node/become-testnet-validator.md:125]: No need to modify this file, I submit another PR to modify the validator part.

Comment (Fynnss) [reply docs/guide/greenfield-blockchain/run-node/become-testnet-validator.md]: made a mistake. will delete it.

File: docs/guide/concept/simple-storage-service.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- A policy associate to a bucket can allow grantee to operate the bucket or the specific objects.
+- A policy associate to a object/group can only allow to operator the object/group.
+
+In the reousrce-based policy, the user can use wildcard characters Greenfield Resource Names(GRNS) and other values to grant permission to a subset of objects. For Example, the user can only allow the grantee to access to the objects that begin with a common prefix or end with a given extension, such as `.html`.
+
+
+## Life Cycle
+To store your data in Greenfield, the user should work with resources known as buckets and objects. A bucket is a container for objects, and an object is a file along with any metadata that describes that file.
+
+To store an object in Greenfield, the user creates a bucket and then uploads the object to that bucket. Once the object is in the bucket, it can be opened, downloaded, and moved. When the user no longer needs an object or a bucket, they can clean up their resources.
```
Comment (guagualvcha) [inline docs/guide/concept/simple-storage-service.md:89]: `opened` and `move` is hard to understand.

File: docs/guide/concept/storage-provider-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+At this stage, SPs must create virtual groups within the Greenfield network to efficiently serve buckets and objects. These virtual groups, resembling disk sectors, allow SPs to manage data storage in a more organized and optimized manner. By associating objects with virtual groups, SPs can limit the range of secondary storage providers responsible for storing object replica data, which enhances data redundancy and resilience.
+
+:::note
+for more information, please see [Virtual Group](../greenfield-blockchain/modules/virtual-group.md#abstract)
+:::
+
+Additionally, SPs are required to provide corresponding stakes for the amount of data they store. This staking mechanism further incentivizes SPs to offer reliable and high-quality services to users. By staking tokens or digital assets, SPs demonstrate their commitment to maintaining a robust and trustworthy network, aligning their interests with the overall security and success of the storage ecosystem.
```
Comment (guagualvcha) [inline docs/guide/concept/storage-provider-lifecycle.md:61]: `By staking tokens or digital assets`, I think it is only BNB in our case.

File: docs/guide/concept/storage-provider-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+Moreover, the creation of virtual groups and staking helps to disentangle the interdependency between buckets/objects and SPs. By doing so, SPs mitigate the need for an extensive volume of transactions when modifying on-chain BucketInfo and ObjectInfo during SP exits and bucket migrations. This leads to more efficient network management and smoother transitions during changes in the network's composition.
+
+As SPs continue to serve user needs and actively participate in network operations, their reputation and service quality become paramount. A positive reputation score is crucial for attracting more users to store their data with a particular SP. Through continuous improvement and adaptation, SPs can enhance their services, increase storage capacity, and maintain a competitive edge in the dynamic decentralized storage market.
```
Comment (guagualvcha) [inline docs/guide/concept/storage-provider-lifecycle.md:65]: I think we dont mention `reputation score` here, because `reputation system` now is a third part implementation, it will cause confusion to mention `reputation score`.

File: docs/guide/concept/storage-provider-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+## Exit
+
+At some point, the SP may choose to voluntarily exit the Greenfield storage network for various reasons. Ensuring a graceful exit process is crucial to ensure a seamless transition of responsibilities and data to other SPs. During the exit process, the SP has the option to withdraw its staked tokens, but this withdrawal may be subject to any penalties or obligations associated with the exit.
```
Comment (guagualvcha) [inline docs/guide/concept/storage-provider-lifecycle.md:69]: `but this withdrawal may be subject to any penalties or obligations associated with the exit`  This is too general, I don't understand it very well, I understand that it has nothing to do with withdraw and penalties.

File: docs/guide/concept/simple-storage-service.md (hunk 2 above)
Comment (Fynnss) [reply docs/guide/concept/simple-storage-service.md:89]: change these to `downloaded and migrated.`

File: docs/guide/concept/storage-provider-lifecycle.md (hunk 3 above)
Comment (Fynnss) [reply docs/guide/concept/storage-provider-lifecycle.md:61]: change to "By staking BNB tokens."

File: docs/guide/concept/storage-provider-lifecycle.md (hunk 5 above)
Comment (Fynnss) [reply docs/guide/concept/storage-provider-lifecycle.md:69]: Change to `After completing the graceful exit, the SP will receive the staking tokens that were initially staked upon joining the network.`

**Review by guagualvcha — APPROVED** [review]

**Review by KeefeL — APPROVED** [review]
